Dockerfile

```diff
@@ -0,0 +1,45 @@
+# based on https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
+FROM node:8-slim
+
+# Install latest chrome dev package and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+  && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+  && apt-get update \
+  && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont \
+  --no-install-recommends \
+  && rm -rf /var/lib/apt/lists/*
+
+# Uncomment to skip the chromium download when installing puppeteer. If you do,
+# you'll need to launch puppeteer with:
+#     browser.launch({executablePath: 'google-chrome-unstable'})
+# ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
+
+
+# copy repo root (without node_modules) to working dir
+# node_modules are excluded by the .dockerignore rules
+# install dependencies
+
+COPY package.json ./
+COPY ./ ./
+RUN yarn install
+
+# # Add user so we don't need --no-sandbox.
+# # same layer as npm install to keep re-chowned files from using up several hundred MBs more space
```

Comment:
There's a couple of commented out sections in this Dockerfile. Are these things that we will need? If not, can we remove?

Reply:
removed